app/serializers/search_result_serializer.rb

```diff
@@ -1,10 +1,6 @@
 class SearchResultSerializer < ActiveModel::Serializer
   embed :id, include: true
-  attributes :id, :priority, :query, :discussion_blurb, :motion_blurb, :comment_blurb
-
-  has_one :discussion
-  has_one :motion, root: :proposals
```

Comment:
Looks really great James.  Thanks for explaining it all too.

On Sat, 31 Oct 2015 at 21:07 James Kiesel <notifications@github.com> wrote:

> In app/serializers/search_result_serializer.rb
> <https://github.com/loomio/loomio/pull/2662#discussion_r43569465>:
>
> > @@ -1,10 +1,6 @@
> >  class SearchResultSerializer < ActiveModel::Serializer
> >    embed :id, include: true
> > -  attributes :id, :priority, :query, :discussion_blurb, :motion_blurb, :comment_blurb
> > -
> > -  has_one :discussion
> > -  has_one :motion, root: :proposals
>
> This serialization was killing us as well, especially given that the
> discussion serializer had some perf issues.
>
> Now we just get a slightly-dumbed-down discussion back as the result, with
> no bells or whistles.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2662/files#r43569465>.
>